### PR TITLE
Adds authors to static pages.

### PR DESCRIPTION
**Pull request by weerd**
### What does this PR do?

This PR adds Authors to static pages (ideally 2 authors max; will be adding validation to Contentful to limit to 2) and also adds some static page title sizing and responsive fixes.

### Any background context you want to provide?

- [Small Screen](https://user-images.githubusercontent.com/105849/45962398-c602e400-bfee-11e8-92e2-0035b8aa6596.png)
- [Medium Screen](https://user-images.githubusercontent.com/105849/45962369-b388aa80-bfee-11e8-934a-ecd1a8d9bd62.png)
- [Large Screen](https://user-images.githubusercontent.com/105849/45962348-a8ce1580-bfee-11e8-9581-4f743a4e98b0.png)

### What are the relevant tickets/cards?

Refs [Pivotal ID #159745156](https://www.pivotaltracker.com/story/show/159745156)
Refs [Pivotal ID #159745284](https://www.pivotaltracker.com/story/show/159745284)

### Checklist

* [x] Added screenshot to PR description of related front-end updates on **small** screens.
* [x] Added screenshot to PR description of related front-end updates on **medium** screens.
* [x] Added screenshot to PR description of related front-end updates on **large** screens.

### CODE REVIEW

File: resources/assets/components/pages/GeneralPage/general-page.scss [hunk 1]
```diff
@@ -41,6 +41,25 @@
...
+  .general-page__authors {
+    display: inline-block;
+    margin: $base-spacing auto 0;
+    text-align: left;
+
+    @include media($large) {
+      margin-top: 36px;
+    }
+
+    .byline {
```
Comment (DFurnes) [inline resources/assets/components/pages/GeneralPage/general-page.scss]: Nested selectors like this make me a little nervous (as opposed to explicitly applying a class to the thing we're styling, and ideally scoping it safely within the module it modifies), but not a blocker!

Comment (weerd) [reply resources/assets/components/pages/GeneralPage/general-page.scss]: it's scoped within the `.general-page__authors` class which is a div containing only the single or multiple byline items. I think this should be cool for now until we change everything to use TailWind!

Comment (DFurnes) [reply resources/assets/components/pages/GeneralPage/general-page.scss]: Yep, I saw it was scoped! My concern is that referencing `.byline` here in `general-page.scss` is not very discoverable, so if we make markup or style changes in `byline.scss` we're less likely to check that the overrides you've added here continue to function.

**Comment by weerd** [comment]
Oh! Gotcha, yeah, good call.

File: resources/assets/components/pages/GeneralPage/general-page.scss (hunk 1 above)
Comment (weerd) [reply resources/assets/components/pages/GeneralPage/general-page.scss]: Oh! Gotcha, yeah, that makes sense.

**Pull request merged**